### PR TITLE
owmods-cli: 0.11.2 -> 0.11.3

### PR DESCRIPTION
**Pull request by loco-choco**
## Description of changes
Updated owmods-cli to version 0.11.3, check release notes in [here](https://github.com/ow-mods/ow-mod-man/releases/tag/cli_v0.11.3).

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by loco-choco** [comment]
Moved owmods-cli to an inner folder, as once owmods-gui is added, I think it would make more sense to have them both in the same place, as they usually update together.

**Comment by NickCao** [comment]
Please split the update script changes into it's own commit, also since we are moving everything into the by-name hierarchy soonish, it makes little sense for that additional folder.

**Comment by NickCao** [comment]
Also try `passthru.updateScript = nix-update-script { };`, it should be able to automatically handle the lockfile.

File: pkgs/applications/misc/owmods-cli/update.sh
Comment (NickCao) [inline pkgs/applications/misc/owmods-cli/update.sh:1]: This file is no longer needed.

**Review by NickCao — APPROVED** [review]
diff LGTM

**Pull request merged**